project.yaml

```diff
@@ -535,3 +519,23 @@ actions:
     outputs:
       moderately_sensitive:
         notebook: output/report/report_weekly.html
+
+
+  ### Actions for paper ###
+
+  ### RUN LOCALLY: use functions not currently available on OS image ###
```

Comment:
Like floats, Int32/Int64 allow for nan values. So it would be nicer if we could specify Int32/Int64 so that we don't have to remember to round later when we want to display counts.  
Some numpy functions (like sqrt) in the version of the image cannot handle Int32/Int64s with nans (up-to-date numpy is fine)
I'll change the read_csv to still treat numerator and denominator as float, and it should run on the image.